modules/mono/csharp_script.cpp

```diff
@@ -2955,13 +2954,24 @@ void CSharpScript::initialize_for_managed_type(Ref<CSharpScript> p_script, GDMon
 
 	CRASH_COND(p_script->native == nullptr);
 
+	p_script->valid = true;
+
+	update_info_from_mono(p_script);
+
+#ifdef TOOLS_ENABLED
+	p_script->_update_member_info_no_exports();
+#endif
+}
+
+// Extract information about the script using the mono class.
+void CSharpScript::update_info_from_mono(Ref<CSharpScript> p_script) {
```

Comment:
Can you change the function name to something like `update_script_class_info`?

Reply:
Done. https://github.com/godotengine/godot/compare/212d88a..e4d20ae